src/App.js

```diff
@@ -1,9 +1,12 @@
 import React from 'react';
+import ToDoList from './Components/ToDoList';
+import Todo from './Components/Todo';
 
 const App = () => (
   <div>
-    <h2>Todo App</h2>
-
+    <h1>Todo App</h1>        
+  <ToDoList />
+    <Todo />    
```

Comment:
We don't need to `render` Todo here, we only ever want to render Todo when we create a new Todo